src/main/scala/faces/apps/ControlledFaces.scala

```diff
@@ -106,8 +106,8 @@ object ControlledFaces extends App {
                 // write images and their parameters
                 println(s"Generating \t ID:$id \t Sample:$n")
                 for((img, postifx) <- imageData) {
-                  helpers.writeRenderParametersAndLandmarks(centered, id, n)
-                  helpers.writeImg(img, id, n, postifx)
+                  helpers.writeRenderParametersAndLandmarks(centered, id, n, img)
+                  helpers.writeImg(img, id, n, postifx, img)
```

Comment:
i don't see why this methods should get the image twice

Reply:
perhaps set a defaultMask for writeImg?

Reply:
please state in the readme, that the occlusions are only supported in the random mode